### ISSUE TITLE
Make does not seem to work

### ISSUE THREAD

**Issue by vandan-revanur** [educ8s/Raylib-CPP-Starter-Template-for-VSCODE#1]
Here is the error stack trace :

```
make game
make[1]: *** No rule to make target `main.c', needed by `game'.  Stop.
make: *** [all] Error 2

 *  The terminal process "/bin/zsh '-c', ''make''" failed to launch (exit code: 2). 
 *  Terminal will be reused by tasks, press any key to close it. 
 * 
 ```

OS specifications:

```
OS: OSX
6-Core Intel Core i7
Processor Speed:	2,6 GHz
Number of Processors:	1
Total Number of Cores:	6
Memory:	32 GB
```

**Comment by vandan-revanur** [comment]
I have figured out the fix. I will make a PR.

**Comment by vandan-revanur** [comment]
Fixed in PR #2
